plan/plan.go

```diff
@@ -701,27 +708,31 @@ func (p *Plan) buildEnvs(conf *v2.Config) (map[string]Env, error) {
 			componentPlan.PathToRepoRoot = "../../../../"
 			componentPlan.CdktfDependencies = map[string]string{
 				// from packages/cdktf-fogg-constructs
-				"@vincenthsh/cdktf-fogg-helpers": "^1.0.0",
-				"@cdktf/provider-aws":            "^19.44.0", // @vincenthsh/cdktf-fogg-helpers peer dependency
-				"@cdktf/provider-cloudflare":     "^11.16.0", // @vincenthsh/cdktf-fogg-helpers peer dependency
-				"@cdktf/provider-datadog":        "^11.8.0",  // @vincenthsh/cdktf-fogg-helpers peer dependency
-				"cdktf":                          "^0.20.8",
+				"@vincenthsh/cdktf-fogg-helpers": "^1.3.0",
```

Comment:
already pointing to the new `minor` release after this PR is merged and released.